assets/js/script-fav.js

```diff
@@ -1,9 +1,36 @@
 $(function () {
-	var storedCocktails = JSON.parse(localStorage.getItem("storedCocktails"));
-	storedCocktails.map((cocktail) => {
-		console.log(cocktail);
-		var favouriteDrinkItem =
-			"<button class='cocktailFavDrinksButton'>" + cocktail + "</button>";
-		$("#cocktailFavDiv").append(favouriteDrinkItem);
-	});
+	// var removeFavCocktail = $(".removeFavCocktail");
+	var favouriteDrinkItem;
+	var storedCocktails;
```

Comment:
As far as I'm aware, global scope variables are really not good practice, best avoided however possible within reason.